app/helpers/application_helper.rb

```diff
@@ -1,2 +1,6 @@
 module ApplicationHelper
+  def full_title page_title
+    base_title = "Ruby on Rails Tutorial Sample App"
```

Comment:
"Ruby on Rails Tutorial Sample App" you should add to setting.yml and call it by `Settings....`

Reply:
may I know more details? or any references?